src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -361,8 +361,9 @@ bool
 CrashInfo::UnwindAllThreads(IXCLRDataProcess* pClrDataProcess)
 {
     ReleaseHolder<ISOSDacInterface> pSos = nullptr;
-    pClrDataProcess->QueryInterface(__uuidof(ISOSDacInterface), (void**)&pSos);
-
+    if (pClrDataProcess != nullptr) {
+        pClrDataProcess->QueryInterface(__uuidof(ISOSDacInterface), (void**)&pSos);
```

Comment:
I wonder if this logic should be inside the UnwindThread. We already always do a null check in there and we only use this interface in case there's an exception. This is pretty cheap, so I don't mind it.

Reply:
I'll clean that up next time.